src/components/choice-tabs/ChoiceTabs.scss

```
@@ -0,0 +1,268 @@
+.choice-tabs__icon-wrap {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 28px;
+  height: 28px;
+  flex: 0 0 28px;
+  border-radius: 8px;
+  border: var(--home-features-item-bg-border, 1px solid rgba(17, 17, 19, 0.08));
+  background: var(--home-features-item-action-btn-bg, #fff);
+  backdrop-filter: blur(16px);
+  // Nudge down 1px to optically center with first text line
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)